### ISSUE TITLE
Add a11y ids and labels to all location view elements

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#19976]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-9041)

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Hello, are there any specific steps for this issue to be checked?
If it’s added to all location view elements and the toolbar redesign is not fully implemented I assume this should be checked only after full implementation.

Winnie Teichmann Sorin Paraipan

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

Andrei Bodea the search engine image, text field and the clear button were given a11y ids and labels in this task - however all buttons/elements in the address and navigation toolbar should have a11y ids.

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v9000 (46300) with iPhone 15 Pro (17.5).

Search engine image ✅VerifiedText Field  ✅VerifiedClear button  ✅VerifiedAdded a new bug ( https://github.com/mozilla-mobile/firefox-ios/issues/22507 ).

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v133 (46984) with iPhone 15 Pro (18.2).
